assets/js/analytics.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Analytics tracking
+ *
+ * @module analytics
+ */
+
+
+/**
+ * Event name (eventAction) to selector context (eventCategory)
+ * @constant
+ * @type {Object<String.String>}
+ */
+const events = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).